setup.py

```diff
@@ -4,38 +4,58 @@
 import torch
 import torch.cuda
 from setuptools import find_packages, setup
-from torch.utils.cpp_extension import (CUDA_HOME, BuildExtension, CppExtension,
-                                       CUDAExtension)
+from torch.utils.cpp_extension import (
+    CUDA_HOME,
+    BuildExtension,
+    CppExtension,
+    CUDAExtension,
+)
+
+# from torchsparse import __version__
 
-from torchsparse import __version__
+version_file = open("./torchsparse/version.py")
```

Comment:
Ok - it looks like this is the line where [they parse the `__version__` from `torchsparse/version.py`](https://github.com/ZendarInc/torchsparse/blob/mk/v2.1.0/torchsparse/version.py)

No idea why they're reading the python file as a text file and parsing it by hand - their `master-v2.2.0` branch does as you do here, but [the `master` branch (which hasn't been touched for years) does it a more sane way](https://github.com/ZendarInc/torchsparse/blob/1a10fda15098f3bf4fa2d01f8bee53e85762abcf/setup.py#L10).


-----------------------------------------------

Anyways, regardless of how the file-reading is done, can you make sure that the version declared in `torchsparse/version.py` is what's correctly getting allocated to the built wheel and saved into the filename?  That's how we'll be versioning `torchsparse` as we use it in `RadarProcessor`.

Reply:
Are you sure you can import `torchsparse` before the package has been built? In general that's not a safe assumption.

Reply:
ah - great - yeah @dranjan maybe that's why they're parsing the `version.py` file as a text file :heavy_check_mark:

Reply:
okay I do not think this is an issue. The built package has the correct version.
```
iPython 3.11.7 (main, Dec  8 2023, 18:56:57) [GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torchsparse
>>> torchsparse.__version__
'2.1.0'
```

Reply:
Great - thanks for checking.

Then please just ensure you follow [the directions to upload the built wheel](https://github.com/ZendarInc/torchsparse?tab=readme-ov-file#rolling-out-updates-at-zendar) once this is merged, including the new version in the github release, and then the versioning should all work out!